src/lib/cli/keyfctl.js

```diff
@@ -89,18 +89,17 @@ capitano.command({
   }],
   action: (params, options) => {
     if (options.verbose) console.error(options)
-
     const {
       deploy,
       write,
       verbose,
-      keyframe = './keyframe.yml',
+      keyframe = ['./keyframe.yml'],
```

Comment:
This is setting the default. If someone passes the `-k file.yml` argument when they run it, `keyframe` will be set to a string, not an array. If it's important that it be an array, you'll need to ensure it afterwards.

Reply:
I missed that! I tested the default and multiple -k arguments (ie `keyfctl generate -k keyframeCore.yml -k keyframeStaging.yml`). Will fix :+1: